tests/cypress/fixtures/create-interface.py

```diff
@@ -0,0 +1,199 @@
+"""TITLE: Layer on Substrate."""
```

Comment:
We should get this from github

Reply:
To name files in plain english? Can we use whitespaces tho?
Or should we do `Place%20Layer%20on%20substrate.py`?

Reply:
I mean, we should get the file itself from api-examples to have one source of truth, not store a copy here

Reply:
Ah, the test fixture. Got it